feature_engine/feature_selection.py

```diff
@@ -88,3 +89,138 @@ def transform(self, X):
         return X
 
 
+class SelectFeatures(TransformerMixin, BaseEstimator):
+    """
+    SelectFeatures()
+
+    Drops constant and quasi constant variables from a dataframe.
+    Constant variables are those variables which show only one value for all the observations.
+    By default, all constant variables are dropped.
+    Quasi-constant variables are those variables which are almost constant. In other words, those features have the same
+    value for large subset of total observations.
+    This transformer works for both, numerical and categorical variables.
+
+    Parameters
+    ----------
+
+    drop_quasi_constant_features : bool, default=False
+        Whether to drop quasi constant features
+
+    quasi_constant_threshold: float, default=None
+        When drop_quasi_constant_features is True, features showing quasi_constant_threshold percentage of values
+        will be dropped
+
+    Attributes
+    ----------
+
+    constant_features_: list
+        List of constant features in a dataframe
+
+    quasi_constant_features_: list
+        List of quasi constant features in a dataframe
+
+    selected_features_: list
+        List of selected features which are non-constant and non-quasi constant
+
+    """
+    def __init__(self, drop_quasi_constant_features=False, quasi_constant_threshold=None):
+
+        self.drop_quasi_constant_features = drop_quasi_constant_features
+        self.quasi_constant_threshold = quasi_constant_threshold
+
+        self.constant_features_ = []
+        self.selected_features_ = []
+        self.quasi_constant_features_ = []
+        self._input_shape_ = ()
+
+        if not self.drop_quasi_constant_features and self.quasi_constant_threshold is not None:
+            warnings.warn(
+                'Setting a quasi_constant_threshold has no effect since drop_quasi_constant_features is False. You '
+                'should leave quasi_constant_threshold to its default (None), or set drop_quasi_constant_features=True.'
+                , FutureWarning
+            )
```

Comment:
please change this as per using only a threshold parameter. Check that threshold takes values between 0 and 1

Reply:
Yes checking it here, will change the name to threshold
``` 
if self.quasi_constant_threshold <= 0 or self.quasi_constant_threshold >= 1:
    raise ValueError("Value of threshold should be between 0 and 1")
```

Reply:
Something like this: https://github.com/solegalli/feature_engine/blob/master/feature_engine/categorical_encoders.py#L714